api/proto/relay/relay.proto

```diff
@@ -9,43 +9,23 @@ option go_package = "github.com/Layr-Labs/eigenda/api/grpc/relay";
 
 // Relay is a service that provides access to public relay functionality.
 service Relay {
```

Comment:
This change looks good to me. Do you want to have a PR with this change and merge it while working on the relay? 
This could unblock node endpoint implementation.

Reply:
Should we update `GetChunksRequest` to allow ranges as well?

Reply:
Thanks! Can you rebase this PR onto master?

Reply:
This branch is currently up to date with master.